.github/scripts/download_pretrained.py

```diff
@@ -52,7 +52,7 @@ def get_model_name_and_weights_from_config(
             if model_name not in model_class_dict:
                 raise KeyError(
                     f"'{model_name}' not a valid model name. Choose from "
-                    f"{str(list(model_class_dict.keys()))} or create"
+                    f"{list(model_class_dict.keys())!s} or create"
```

Comment:
Why was this change required?

Reply:
Oh my bad, that was fixed automatically by ruff, missed this when reverting the ruff upgrade. Thanks